### DISCUSSION TITLE
Puzzlehash used by MadMax

### DISCUSSION THREAD

**Discussion by porkopops** [madMAx43v3r/chia-plotter#756] [Q&A]
Madmax shows a different puzzlehash than expected.

I've installed the updated Chia 1.2.0 on the Farmer and joined a pool using the GUI. I then used 'chia plotnft show' to obtain the contract/singleton address.

During the process above I was presented with a puzzlehash. Although I can't remember which part of it, I noted it down at the time, (I expect the GUI part of the process).

I've installed the updated Chia 1.2.0 on the Harvesters and closed it down.

Updated the CMD used to run Madmax replacing -p with the -c contract_address.

However, Madmax (on the Harvesters) is showing a different puzzlehash than I originally saw when going through the process above on the Farmer. Will this cause an issue?

**Comment by madMAx43v3r** [comment]
puzzle hash is the hex value, the contract address is a bech32 encoding of that value, that's why they look different but are the same value

**Comment by madMAx43v3r** [comment]
you can convert between the two formats here: https://www.chiaexplorer.com/tools/address-puzzlehash-converter

in practice you don't need to however, the only reason I show the puzzle hash to verify that the bech32 decoding was correct

**Reply by porkopops** [reply]
I see. Many thanks @madMAx43v3r
